life50x50/Form1.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+
+namespace life50x50
+{
+    public partial class Form1 : Form
+    {
+        private bool[,] gameboard = new bool[50, 50];
+        private bool gamestarted = false;
+        private int stage;
+        private int changes;
+        private int totalAlive;
+        private Timer timer = new Timer();
+
+
+        Pen myPen = new Pen(Color.Gray);
+        Brush myBrush = new SolidBrush(Color.Black);
+        Graphics g = null;
+
+        public Form1()
+        {
+            InitializeComponent();
+            timer.Tick += new EventHandler(TimerEventProcessor);
+            timer.Interval = 1000;
+            menuStrip1.Items[1].Visible = false;
+            prepgame();
+        }
+
+        private void prepgame()
+        {
+            stage = 1;
+            totalAlive = 0;
+            changes = 0;
+            toolStripStatusLabel1.Text = "Поколение "+stage.ToString();
+            for (int i = 0; i < 50; i++)
+            {
+                for (int j = 0; j < 50; j++)
+                {
+                    gameboard[i, j] = false;
+                }
+            }

```

Comment:
Пишите код компактнее.
```suggestion
            for (int i = 0; i < 50; i++)
                for (int j = 0; j < 50; j++)
                    gameboard[i, j] = false;


```